library/alloc/src/string.rs

```diff
@@ -2530,12 +2531,51 @@ macro_rules! impl_eq {
 
 impl_eq! { String, str }
 impl_eq! { String, &'a str }
+impl_eq! { String, &String }
+impl_eq! { &String, str }
```

Comment:
These two impls end up with incorrect stability attributes, which means Clippy msrv autodetection can't be added later.

Reply:
That's fixable, but is that worth complicating the macro for? There are many other examples of macro-generated instances with the same discrepancy.